### ISSUE TITLE
[Feature Request] Single Instance

### ISSUE THREAD

**Issue by OLLI-S** [tjackenpacken/taskbar-groups#64]
When I edit a group, I can switch to the Taskbar Groups window and click on "Edit" again (here I have two instances of the same edit window opened, so I edit the same group in two different widows, what could cause problems).
When I edit the same group a second time, switch to the edit window that is already opened.

I also saw that I can open two instances of Taskbar Groups.
If this is not intended, you should also prohibit this.

**Comment by tjackenpacken** [comment]
You're right. I have noticed this aswell and agree that it's not a great behaviour. 

My plan is to get rid of the group edit as a separate form/window, and instead have it as an embedded panel/tab in the client.

**Comment by OLLI-S** [comment]
This sounds like a perfect solution!

**Comment by PikeNote** [comment]
Will push in temporary fix until that is embedded which I assume may take a bit
